.goreleaser.yml

```diff
@@ -124,7 +147,7 @@ release:
     #### Linux
 
     - 📦 [step_linux_{{ .Version }}_amd64.tar.gz](https://dl.step.sm/cli/{{ .Tag }}/step_linux_{{ .Version }}_amd64.tar.gz)
-    - 📦 [step-cli_{{ .Env.DEB_VERSION }}_amd64.deb](https://dl.step.sm/cli/{{ .Tag }}/step-cli_{{ .Env.DEB_VERSION }}_amd64.deb)
```

Comment:
Why are we changing this? Originally I used DEB_VERSION instead of VERSION because our release candidate versions have a dash which is generally not supported in debian package versions. So DEB_VERSION would be the version but with the dashes replaced by another character. 

If we no longer need to worry about dashes then we should remove the presence of `DEB_VERSION` entirely. If we do need to worry about them, then we should switch back to using `DEB_VERSION`. We should test with a release candidate.

Reply:
So, ultimately the way this worked out with GoReleaser is, the metadata will have `~` instead of `-`, but the filename will have the `-`. And I think that's OK, but if we need something different we can specify a custom filename in the the `nfpms` Linux packager config above